pkg/git/libgit2/managed/transport_test.go

```diff
@@ -66,7 +109,7 @@ func TestKnownHostsCallback(t *testing.T) {
 			knownHosts:   []byte(knownHostsFixture),
 			hostkey:      git2go.HostkeyCertificate{Kind: git2go.HostkeySHA256, HashSHA256: sha256Fingerprint("ROQFvPThGrW4RuWLoL9tq9I9zJ42fK4XywyRtbOz/EQ")},
 			expectedHost: "github.com",
-			want:         fmt.Errorf("hostkey could not be verified"),
+			want:         fmt.Errorf("no entries in known_hosts matches host 'github.com' with fingerprint 'ROQFvPThGrW4RuWLoL9tq9I9zJ42fK4XywyRtbOz/EQ='"),
```

Comment:
Pointing this out just to highlight how this looks like when printing the fingerprint in the error, so that we can discuss and think if there's a better way to express this error.